scale-testing/pfioh_scale/test_pfioh_request.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+import sys, os
+sys.path.append('/home/ckaubisch/ChRIS-E2E/scale-testing')
+from test_request import test_request
+from scale_threads import scale_threads
+import time
+import threading
+import matplotlib.pyplot as plt
+import numpy as np
+from subprocess import Popen, PIPE
+import test_setup
+import uuid
+
+
+"""
+Performs scalability and performance tests by making increasingly more concurrent push/pull requests to pfioh and measuring resultant resource utilization and success rate
+"""
+
+RANGE = 50                                 # Number of concurrent requests to scale up to
+CAPTURE = True                            # Dictates whether or not to capture CPU and memory utilization
```

Comment:
Formatting is misaligned on the comment.

Also these should be moved to config.  You shouldn't have to modify code to changes these sorts of things.